lib/packages/crypto/src/decryptData.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import * as CryptoJS from 'crypto-js';
-import {inflate} from 'pako';
+import {inflate} from 'pako/lib/inflate';
```

Comment:
this is cool. we can do treeshaking here. crypto is a large lib, but not that large. I was worried that the zip stuff from pako would bloat the code base tremendously... finally less than I expected